roles/README.adoc

```diff
@@ -1,63 +1,36 @@
-  This covers several related use cases:
-** Users that want to manage multiple major releases of the system simultaneously with a single playbook.
-** Users that want to migrate to a new version of the system without changing their automation (playbook).
-** Users who want to switch to a different provider in the same version of the OS (like switching from ntpd to chrony to RHEL 7) and keep the same playbook.
-
-Designing the interface in the latter case is difficult because it has to be sufficiently abstract to cover different providers.
-We, for example, do not provide an email role in the Linux System Roles project, only a postfix role, because the underlying implementations (sendmail, postfix) were deemed to be too divergent.
-Generally, an abstract interface should be something that should be always aimed for though, especially if there are multiple providers in use already, and in
-particular when the default provider is changing or is known to be likely to change in the next major releases.
+Explanations:: Design roles focused on the functionality provided, not the software implementation.
+
+Rationale:: Try to design roles focused on the functionality, not on the software implementation behind it. This will help abstracting differences between different providers, and help the user to focus on the functionality, not on technical details.
+
+Examples:: For an example, designing a role to implement an NTP configuration on a server would be a role.  The role internally would have the logic to decide whether to use ntpd, chronyd, and the ntp site configurations.
+However, when the underlying implementations become too divergent, for example implementing an email server with postfix or sendmail, then separate roles are encouraged.
+
 ====
 
-== Basics
+=== Role Structure
 [%collapsible]
 ====
-* Every repository in the AGP-roles namespace should be a valid Ansible Galaxy compatible role with the exception of any whose names begin with "meta_", such as this one.
-* New roles should be initiated in line with the skeleton directory, which has standard boilerplate code for a Galaxy-compatible Ansible role and some enforcement around these standards
-* Use https://semver.org/[semantic versioning] for Git release tags.
-  Use 0.y.z before the role is declared stable (interface-wise).
-  Although it has not been a problem so far for linux system roles, since they use strict X.Y.Z versioning, you should be aware that there are some
-https://github.com/ansible/ansible/issues/67512[restrictions] for Ansible
-Galaxy and Automation Hub.
-  The versioning must be in strict X.Y.Z[ab][W] format, where X, Y, and Z are integers.
-====
-
-== Interface design considerations
+Explanations:: New roles should be initiated in line, with the skeleton directory, which has standard boilerplate code for a Galaxy-compatible Ansible role and some enforcement around these standards
```

Comment:
Don't we have a rule about deleting empty files/directories somewhere? I'm not 100% sure r/n @ericzolf do you happen to know?